Cogs.Publishers/JsonSimpleConverter.cs

```diff
@@ -31,7 +31,7 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
                 obj.Add(new JProperty("dateTime", 
                     new JObject(
                         new JProperty("type", "string"), 
-                        new JProperty("format", "date-time"))));
+                        new JProperty("pattern", @"^[0-9][0-9][0-9][0-9]-(0?[1-9]|1[0-2])-(0?[1-9]|[12][0-9]|3[01])([A-Z])(00|0[0-9]|1[0-9]|2[0-3]):([0-9]|[0-5][0-9]):([0-9]|[0-5][0-9])(\+)(00|0[0-9]|1[0-9]|2[0-3]):([0-9]|[0-5][0-9]):([0-9]|[0-5][0-9])$"))));
```

Comment:
The build in date-time format, only accept 
"2012-04-23T18:25:43.511Z"     
"2012-04-21T18:25:43-05:00"
But not
"2012-04-21T18:25:43-05:00:00"
Since the offset comes in the form of  XX:XX:XX, it is hard to parse these value and convert it back and forth.